### DISCUSSION TITLE
Welcome, Lucas!

### DISCUSSION THREAD

**Discussion by lucas-0liveira** [ParanoidUser/yolo#1177] [Subscribe]
My name is Lucas and it will be a pleasure to share my knowledge in IT 😁

**Comment by ParanoidUser** [comment]
Ahoy @lucas-0liveira, my apologies for the long replies to your recent PRs. To get a Yolo badge, simply follow the steps in https://github.com/ParanoidUser/yolo/discussions/18.

**Note:** Don't forget to assign a reviewer on step 4. 🔭

**Another note:** You should receive a repo collaborator invite that allows you to merge PRs (steps 5-6).

Let me know if you have any questions. Welcome aboard! 🚢⭐

**Reply by lucas-0liveira** [reply]
Thank you for the answer @ParanoidUser , I will follow your steps and if I have any questions I will tag you, Thanks!
